docs/content/config/environment.md

```diff
@@ -80,6 +80,15 @@ Enable or disable Rspamd.
 - **0** => disabled
 - 1 => enabled
 
+##### ENABLE_REDIS
+
+Enables or disable a freestanding Redis instance inside the container. The default value is the value of [`ENABLE_RSPAMD`](#enable_rspamd), i.e. if you set `ENABLE_RSPAMD=X` this variable will also default to `X` if not specified.
+
+This way, you can enable Rspamd (`ENABLE_RSPAMD=1`) but run a dedicated Redis instance in another container (also set `ENABLE_REDIS=0`). Note that you will then need to provide `/etc/rspamd/local.d/redis.conf` yourself!
```

Comment:
Revised wording and included an admonition for example config required and link to relevant upstream docs.

```suggestion
Explicit control over running a Redis instance within the container. By default, this value will match what is set for [`ENABLE_RSPAMD`](#enable_rspamd).

The purpose of this setting is to opt-out of starting an internal Redis instance when enabling Rspamd, replacing it with your own external instance.

??? note Configuring rspamd for an external Redis instance

    You will need to [provide configuration][config-rspamd-redis] at `/etc/rspamd/local.d/redis.conf` similar to:
    
    ```
    servers = "redis.example.test:6379";
    expand_keys = true;
    ```

[config-rspamd-redis]: https://rspamd.com/doc/configuration/redis.html
```